examples/ui.apps/src/main/content/jcr_root/apps/cif-components-examples/clientlibs/venia-theme/cif-demo.css

```diff
@@ -22,6 +22,12 @@
     border: none !important;
 }
 
+.cif-demo .productteaser .item__name, .cif-demo .productteaser .price {
```

Comment:
Can you update the Venia style first and overwrite whats needed afterwards? (may be done manually)